ci-operator/step-registry/openstack/provision/performanceprofile/openstack-provision-performanceprofile-commands.sh

```diff
@@ -91,32 +91,41 @@ HUGEPAGES="${HUGEPAGES:-1}"
 CPU_ISOLATED="${CPU_ISOLATED:-2-7}"
 CPU_RESERVED="${CPU_RESERVED:-0-1}"
 
-PAO_NAMESPACE=$(
-    oc create -f - -o jsonpath='{.metadata.name}' <<EOF
+oc_version=$(oc version -o json | jq -r '.openshiftVersion')
+# Once 4.10 is GA, we need to bump it to 4.11 and so on.
+if [[ "${oc_version}" == *"4.10"* ]]; then
```

Comment:
doesn't the same apply for previous versions?

Reply:
no, in previous versions we want to install the operator from the Marketplace to test what customers have (I think, for now). We can probably change it later if we want to catch problems earlier than that, by installing it from source even on 4.9 etc. WDYT?